packages/bootstrap/extra/dcos_internal_utils/cli.py

```diff
@@ -44,6 +44,16 @@ def dcos_metrics_agent(b, opts):
     b.cluster_id('/var/lib/dcos/cluster-id', readonly=True)
 
 
+@check_root
+def dcos_telegraf_master(b, opts):
+    b.cluster_id('/var/lib/dcos/cluster-id')
+
+
+@check_root
+def dcos_telegraf_agent(b, opts):
+    b.cluster_id('/var/lib/dcos/cluster-id', readonly=True)
```

Comment:
We have now learned that *this* is the critical part of the change (creation of the cluster ID file on *agent* nodes).